src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss

```diff
@@ -1,5 +1,4 @@
 @import "scss/constants.scss";
-
```

Comment:
Probably good to keep this space (this same comment goes for all of the lines where whitespace was removed)

Reply:
I'll fix it, but I didn't actually delete the whitespace? Do you know how that could have happened?

Reply:
It's possible that your code editor/IDE is configured to automatically reformat things in a certain style. That's usually where these issues tend to come from.